templates/Elegant/compare.tmpl

```diff
@@ -35,7 +35,8 @@
     </div>
     [websvn-test:warning]
       <div id="warning">[websvn:warning]</div>
-    [websvn-else]
+    [websvn-endtest]
+    [websvn-test:success]
```

Comment:
Are you sure the success-test is what we want? Look at the BlueGrey-template, it seems to be ONLY used for adding some navigation-block. OTOH, your success-checks in the other templates seem to span ALL content instead. This seems to defeat the purpose from showing content as often as possible. Even if `success` is `false`, `comp.php` continues to work:

				// Check for error messages
				if (strncmp(trim($line), 'svn: ', 5) == 0) {
					$listing[$index++]['info'] = urldecode($line);
					$vars['success'] = false;
					continue;
				}

So I suggest removing `[websvn-test:success]` and the corresponding closing statement again OR make it only work like in `BlueGrey`, for some navigation. But seems easier to me to remove that special treatment from `BlueGrey` as well instead. We don't need to care if that special navigation is hidden sometimes or not in my opinion.

Reply:
Ahh okay this is an issue with my understanding of the success block. Then i need to remove one of the `[websvn-endtest]` from bottom as well. Let me do that.